scripts/globals/abyssea.lua

```diff
@@ -1206,14 +1206,11 @@ xi.abyssea.searingWardTimer = function(player)
     local tetherTimer = player:getLocalVar('tetherTimer')
 
     if tetherTimer > 1 then
-        if tetherTimer == 11 or tetherTimer <= 6 then
+        player:setLocalVar('tetherTimer', tetherTimer - 1)
```

Comment:
I moved this to be the first thing we did so it matched the way the logic in the `==1` case worked, which was to first and foremost update the tetherTimer value, which seemed most correct to me.

Happy to move it back if there is another reason for keeping it as it was.